### ISSUE TITLE
Overlay text on video container does not have sufficient contrast

### ISSUE THREAD

**Issue by cemms1** [guardian/dotcom-rendering#10837]
![image](https://github.com/guardian/dotcom-rendering/assets/43961396/5ffbeec2-d002-40c4-8d95-3ca4ab7a87e4)

[Reported by CP via email](https://groups.google.com/a/guardian.co.uk/g/dotcom.platform/c/3BVV23L7aT4)

> "I find this red kicker on (almost) black in the video overlay in the Video container incredibly hard to read, I wonder if it's intended/whether there are accessibility considerations we're missing or it's just me..."

**Comment by benwuersching** [comment]
Please replace the kicker colour with [pillar].500, so in this case news.500. Do this for all video overlays.

![fix](https://github.com/guardian/dotcom-rendering/assets/23035818/c38cbbdf-451a-4032-b5d8-7f900bc50a90)
